libs/components/ag-grid/src/lib/modules/ag-grid/types/autocomplete-properties.ts

```diff
@@ -2,20 +2,62 @@ import { TemplateRef } from '@angular/core';
 import { SkyAutocompleteSelectionChange } from '@skyux/lookup';
 
 export interface SkyAgGridAutocompleteProperties {
+  /**
+   * Allows the user to specify arbitrary values not in the search results.
+   * @default false
+   */
   allowAnyValue?: boolean;
+  /**
+   * The static data source for the autocomplete cell to search when users enter text. For a dynamic data source such as an array that changes due to server calls, use search instead.
+   */
   data?: unknown[];
+  /**
+   * How many milliseconds to wait before searching while users enter text in the autocomplete field.
+   * @default 0
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Highlights the search text in each search result. Set this to false when your search finds results that are not exact text matches, e.g. returning "Bob" for the term "Robert."
+   * @default true
+   */
   highlightSearchText?: boolean;
+  /**
+   * The object properties to search.
+   * @default ["name"]
+   */
   propertiesToSearch?: string[];
+  /**
+   * The function that dynamically manages the data source when users change the text in the autocomplete cell. The search function must return an array or a promise of an array.
+   */
   search?: (
     searchText: string,
     data?: unknown[],
   ) => unknown[] | Promise<unknown[]>;
+  /**
+   * The array of functions to call against each search result in order to filter the search results when using the default search function. When using the search property to specify a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```

Comment:
```suggestion
   * The array of functions to call against each search result. This filters the search results when using the default search function. When the `search` property specifies a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```
I think the first sentence is hard to follow because of the long string of prepositional phrases, and I think breaking it into 2 sentences makes it easier to understand without changing the meaning. I also think we can tighten up the opening clause of the second sentence without changing the meaning. These are minor edits that we can disregard if it's going to make this description out of sync with another version elsewhere, but I can also follow up and update any other instance.